JM/CF/Scripts/1_Core/CommunityFramework/IO/CF_BinaryWriter.c

```diff
@@ -94,15 +94,15 @@ class CF_BinaryWriter : CF_IO
 		
 		for (int i = 0; i < value.Length(); i++)
 		{
-			m_Stream.AppendCurrent(value.Substring(i, 1).Hash());
+			m_Stream.AppendCurrent(value[i].ToAscii());
```

Comment:
Why would @Jacob-Mango have used substring for this before instead of direct array access? I would think this updated code is correct but I'd like to confirm with him.

Reply:
> Get(n) calls substr(1), and [n] is just (simplified) find and replace for Get(n)
So substr(1) would be faster

is what he said to me via Discord. So please undo the Substring replacements @wrdg